### PR TITLE
Adding datadog middleware for metrics

### PR DESCRIPTION
**Pull request by dexter1691**

### CODE REVIEW

File: middleware/metrics/metrics_middleware.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+        if 200 <= response.status_code < 400:
+            statsd.increment(success_metric, tags=tags)
+        else:
+            statsd.increment(unsuccess_metric, tags=tags)
+
+        statsd.increment(count_metric, tags=tags)
+        statsd.histogram(timing_metric, request_time, tags=tags)
+
+        print timing_metric, count_metric, success_metric, unsuccess_metric
```
Comment (deshraj) [inline middleware/metrics/metrics_middleware.py]: Should we just print them or use logging statements?

**Comment by dexter1691** [comment]
PTAL @deshraj @TroJan

File: middleware/metrics/metrics_middleware.py (hunk 1 above)
Comment (dexter1691) [inline middleware/metrics/metrics_middleware.py]: shit. shouldn't have been there. my bad

**Comment by deshraj** [comment]
Also, can you please add `middlewares/metrics` to this location https://github.com/Cloud-CV/EvalAI/blob/master/.coveragerc#L15

**Pull request merged**